shap/plots/_utils.py

```diff
@@ -22,7 +22,7 @@ def convert_ordering(ordering, shap_values):
     if issubclass(type(ordering), OpChain):
         ordering = ordering.apply(Explanation(shap_values))
     if issubclass(type(ordering), Explanation):
-        if "argsort" in [op["name"] for op in ordering.op_history]:
+        if any("argsort" in op.name for op in ordering.op_history):
```

Comment:
The previous implementation checked for a name that was equal to "argsort" rather than contains "argsort", so I think to be consistent with previous behaviour it would be:

```suggestion
        if any(op.name=="argsort" for op in ordering.op_history):
```

Reply:
Thanks, good catch